server/public/styles.css

```diff
@@ -84,5 +84,71 @@ input[type="submit"] {
   max-height: 100%;
   height: 100%;
   border-radius: 8px;
-  cursor: pointer;
+}
+
+.view-profile-name {
+  color: #808080;
+  font-size: 2.25rem;
+  font-weight: 100;
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  padding: 0;
+  width: 16rem;
+  overflow: hidden;
+}
+
+/* ViewingAccount CSS */
+
+.fa-star-holder {
+  font-size: 2.5rem;
+  margin-top: 1.5rem;
+  color: #626262;
+}
+
+.fa-star-style {
+  margin: 0 0.25rem;
+}
+
+.star-div {
+  color: #626262;
+  position: relative;
+}
+
+.happy {
+  color: #34a604;
+}
+
+.content {
+  color: #d7c9b4;
+}
+
+.angry {
+  color: red;
+}
+
+.half-star-happy {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 50%;
+  color: #34a604;
+  overflow: hidden;
+}
+
+.half-star-content {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 50%;
+  color: #d7c9b4;
+  overflow: hidden;
+}
+
+.half-star-angry {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 50%;
+  color: red;
+  overflow: hidden;
```

Comment:
Please take some time to de-duplicate these CSS properties. the only difference is the color.